evm-adapters/src/sputnik/cheatcodes/cheatcode_handler.rs

```diff
@@ -77,6 +77,10 @@ impl<'a, 'b, B: Backend, P: PrecompileSet> SputnikExecutor<CheatcodeStackState<'
         self.handler.config()
     }
 
+    fn expected_revert(&self) -> Option<Vec<u8>> { 
+       self.handler.state().expected_revert.clone()
```

Comment:
you can do this instead to get an option to the reference
```suggestion
       self.handler.state().expected_revert.as_ref()
```

Reply:
Somebody help me 😀 
```
fn expected_revert(&self) -> Option<&[u8]> { 
        let a: &Vec<u8> = self.handler.state().expected_revert.as_ref().unwrap();
        let c: &[u8] = a;
        return Some(c);
     }
```

Reply:
surely there is a better way

Reply:
er, better, but still seems should be simpler
```
fn expected_revert(&self) -> Option<&[u8]> { 
        if let Some(a) = self.handler.state().expected_revert.as_ref() {
            let c: &[u8] = a;
            return Some(c);
        }
        return None
    }
```